core/src/main/java/com/linecorp/armeria/common/http/HttpResponseAggregator.java

```diff
@@ -38,7 +38,7 @@
     public void onNext(HttpObject o) {
         if (o instanceof HttpHeaders) {
             final HttpHeaders headers = (HttpHeaders) o;
-            if (headers.status().codeClass() == HttpStatusClass.INFORMATIONAL) {
+            if (headers.status() != null && headers.status().codeClass() == HttpStatusClass.INFORMATIONAL) {
```

Comment:
Hmm, indeed. I thought it doesn't do any memoization. Extraction doesn't do harm fortunately. :-) Thanks!